src/ngraph/op/fused/gelu_grad.hpp

```diff
@@ -0,0 +1,45 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/util/fused_op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        class GeluGrad : public util::FusedOp
```

Comment:
Should we call ours `GeluBackprop`? This is more consistent with our current naming scheme. (Though actually if I had to start over I might rename everything else to `Grad`.)

Reply:
I also like `Grad` better but we are already using `Backprop` in many places already (I guess thanks to how mkl-dnn names their primitives.)

Reply:
hmmm.. which one is more ngraph - user friendly? I like backprop, since it is more high level than grad, but is that what everyone 'gets' easily?